api/src/pcapi/core/providers/titelive_api.py

```diff
@@ -99,7 +99,8 @@ def get_updated_titelive_pages(
         while has_next_page:
             titelive_json_response = titelive.search_products(self.titelive_base, from_date, page_index)
             titelive_product_page = self.deserialize_titelive_search_json(titelive_json_response)
-            allowed_product_page = self.filter_allowed_products(titelive_product_page)
+            titlive_recent_product_page = self.filter_recent_products(titelive_product_page, from_date)
+            allowed_product_page = self.filter_allowed_products(titlive_recent_product_page)
```

Comment:
```suggestion
            recently_updated_product_page = self.filter_recent_products(titelive_product_page, from_date)
            allowed_product_page = self.filter_allowed_products(recently_updated_product_page)
            allowed_product_page.result = [oeuvre for oeuvre in allowed_product_page.result if oeuvre.article]
```

la suppression des `oeuvre` sans article éligible peut être fait avant le `yield`, du coup on peut le retirer des `filter_allowed_products` comme dans `titelive_music_search.py:46`